src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -2078,13 +2076,13 @@
                 {
                     "type": "bool",
                     "name": "graphicsPipelineLibraryFastLinking",
-                    "value": false,
+                    "value": true,
                     "expose": "DISABLE"
                 },
                 {
                     "type": "bool",
                     "name": "graphicsPipelineLibraryIndependentInterpolationDecoration",
-                    "value": false,
+                    "value": true,
                     "expose": "DISABLE"
```

Comment:
we don't require the graphics pipeline feature (yet) and we've disabled, no need to set it to true